src/unix/linux-core.c

```diff
@@ -90,7 +90,11 @@ int uv__platform_loop_init(uv_loop_t* loop) {
    * a.k.a. Lollipop. Since EPOLL_CLOEXEC is an alias for O_CLOEXEC on all
    * architectures, we just use that instead.
    */
+#if !(defined(__ANDROID_API__) && __ANDROID_API__ < 21)
   fd = epoll_create1(O_CLOEXEC);
+#else
+  fd = -1;
```

Comment:
This won't work because the fallback path a few lines below also checks `errno`.